src/voxel.py

```diff
@@ -189,6 +202,9 @@ def delete_frame(self):
         if self._current_frame > killframe:
             self._current_frame -= 1
 
+    def get_current_frame(self):
+        return self._current_frame
```

Comment:
Will refactor to get_current_frame as getters should be named after field name. Makes for easier searching...